packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -104,6 +112,8 @@ export class BlockList extends Component {
 			shouldShowInsertionPointAfter,
 			marginVertical = styles.defaultBlock.marginTop,
 			marginHorizontal = styles.defaultBlock.marginLeft,
+			horizontalDirection,
+			horizontalAlignment,
```

Comment:
What’s the difference between these 2?

Reply:
`horizontalDirection` is a flag saying that we have block with horizontal orientation. It is used for e.g. rotate the block movers.

`horizontalAlignment` can be passed as  'right, 'center', 'left' and it set the proper alignment for Button inside Buttons block at BlockList level

Reply:
Thanks for an explanation Jakub!

Reply:
@jbinda You do the same for columns but w/o that prop, right? Maybe it should be handled in the same way?

Reply:
Thanks Jakub. It is not obvious at the first glance that horizontalDirection is a boolean. Maybe a rename could help,  something like: `isHorizontal` or `isStackedHorizontally` etc.

Reply:
I will refactor it to `isStackedHorizontally`, thanks for pointing that!

Reply:
Thanks!